src/main/java/com/example/employees/controller/EmployeeController.java

```diff
@@ -0,0 +1,31 @@
+package com.example.employees.controller;
+
+import com.example.employees.model.Employee;
+import com.example.employees.service.EmployeeService;
+import lombok.AllArgsConstructor;
+import org.springframework.web.bind.annotation.*;
+
+import javax.validation.Valid;
+
+@RestController
+@AllArgsConstructor
+public class EmployeeController {
+
+    private final EmployeeService employeeService;
+
+    @GetMapping("employee/{employeeNo}")
+    public Employee getEmployeeByNo(@PathVariable Integer employeeNo) {
+        return employeeService.findEmployeeByNo(employeeNo);
+    }
+
+    @DeleteMapping("employee/{employeeNo}")
+    public String deleteEmployeeByNo(@PathVariable Integer employeeNo) {
+        return employeeService.deleteEmployeeByNo(employeeNo);
+    }
+
+    @PostMapping("employee/")
+    public Employee postEmployee(@RequestBody @Valid Employee employee) {
```

Comment:
for the request parameter, I'd use a different class, instead of an entity.
this way, we'll not mix database annotations with rest validation annitations
plus, we may have different set of fields in two classes. in this case, we don't need to pass `employeeNo`

also, try not to use entities as a response payload